<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Enigne</class>
 <widget class="QWidget" name="Enigne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>674</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>652</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background:transparent;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>50</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/显示_参数.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="enigne_temp">
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/temp.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 50px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>水温</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/显示_参数.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="enigne_pressure">
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 20px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>kPa</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 46px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>油压</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/显示_参数.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="enigne_speed">
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>0000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 20px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>rpm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 50px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>转速</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="sign">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/背景_标题2.png);
font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;</string>
       </property>
       <property name="text">
        <string>发动机</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#btn_back{
background-image: url(:/按键_上一页2.png);
border:none;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="widget_9" native="true">
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="minimumSize">
         <size>
          <width>366</width>
          <height>174</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>366</width>
          <height>174</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/显示_参数_宽2.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="enegin_hours">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 32px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>发动机累计运行时长</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
